WORKSPACE.bazel

```diff
@@ -3,10 +3,22 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 http_archive(
-    name = "ca-certificates",
+    name = "bullseye-ca-certificates",
```

Comment:
I was thinking about this. Is it unusual for us to rename these to indicate they are for test only `test-bullseye-ca-caerts`? Or `example-bullseye-ca-certs`

Reply:
oh yes let's rename them so that they have the example prefix